hack/dev-build-gce.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a convenience script for developing kOps on GCE.
+# It builds the code, including nodeup, and uploads to a custom GCS bucket.
+# It also sets KOPS_STATE_STORE and KOPS_BASE_URL to project-isolated values.
+# To use, source the script.  For example `. hack/dev-build-gce.sh` (note the initial `.`)
+
+# Can't use set -e in a script we want to source
+#set -e
+
+#set -x
+
+# Build and upload to bucket
+UPLOAD_DEST_BUCKET="kops-dev-$(gcloud config get-value project)-${USER}"
+export UPLOAD_DEST=gs://${UPLOAD_DEST_BUCKET}
+gsutil ls "${UPLOAD_DEST}" || gsutil mb "${UPLOAD_DEST}" || return
+make kops-install dev-upload || return
```

Comment:
I find using dev-upload-linux-amd64 saves a lot of time. dev-upload will rebuild and re-upload from scratch on every run.

Reply:
Good tip - changed, and also made sure to set KOPS_ARCH to only be amd64.